main.go

```diff
@@ -22,13 +23,21 @@ func main() {
 	webDir := getEnv("WEB_DIR", "web")
 	siriusURL := getEnv("SIRIUS_URL", "http://localhost:9001")
 
-	templates, err := template.New("").Funcs(map[string]interface{}{
-		"join": func(sep string, items []string) string {
-			return strings.Join(items, sep)
-		},
-	}).ParseGlob(webDir + "/template/*.gotmpl")
-	if err != nil {
-		logger.Fatalln(err)
+	layouts, _ := template.New("").ParseGlob(webDir + "/template/layout/*.gotmpl")
+
+	files, _ := filepath.Glob(webDir + "/template/*.gotmpl")
+	tmpls := map[string]*template.Template{}
+
+	for _, file := range files {
+		logger.Println(file)
+		tmpls[filepath.Base(file)], _ = template.
+			Must(layouts.Clone()).
+			Funcs(map[string]interface{}{
```

Comment:
Is it possible to move `Funcs` to where we parse the layouts? If not you could extract the map to a variable